client/js/game.js

```diff
@@ -12209,7 +12210,7 @@ function Game() {
 
                 if (CQW!==undefined && CQW.lottery!==undefined) {
                     text(ctx,"- Each lottery ticket costs 1 Ascension Sphere",W*0.5-bw*0.5+10,H*0.3,"36px"+FONT,"white","left","middle");
-                    text(ctx,"- You can buy unlimited lottery tickets",W*0.5-bw*0.5+10,H*0.3+20,"36px"+FONT,"white","left","middle");
+                    text(ctx,"- You can buy 1000 lottery tickets; quick buy: ctrl x10, shift x100",W*0.5-bw*0.5+10,H*0.3+20,"36px"+FONT,"white","left","middle");
```

Comment:
current keybindings are as follows:
```suggestion
                    text(ctx,"- You can buy 1000 lottery tickets; quick buy: ctrl x10, alt x100",W*0.5-bw*0.5+10,H*0.3+20,"36px"+FONT,"white","left","middle");
```

Reply:
Oh right ^^